index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <div class=""></div>
+        <div class="">
+            <a href="" class="logo">Alexis UWIMANA</a>
+        <input class="menu-btn" type="checkbox" id="menu-btn" />
+        <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
```

Comment:
- [ ] Please ensure to wrap your hamburger icon inside the `<nav>` tag. This is so as to enhance the semantics of your HTML code.